applications/app/model/InteractivesDotcomRenderingDataObject.scala

```diff
@@ -0,0 +1,180 @@
+package model
+
+import com.gu.commercial.branding.Branding
+import com.gu.commercial.display.AdTargetParam
+import com.gu.contentapi.client.utils.format.{ArticleDesign, NewsPillar, StandardDisplay}
+import common.commercial.EditionCommercialProperties
+import common.{Edition, Localisation}
+import conf.Configuration
+import model.dotcomrendering.DotcomRenderingUtils.{designTypeAsString, findPillar, makeMatchUrl}
+import model.dotcomrendering.pageElements.TextCleaner
+import model.dotcomrendering.{Author, DotcomRenderingDataModel, DotcomRenderingUtils, PageFooter, PageType}
+import navigation.{FooterLinks, Nav, NavLink, ReaderRevenueLink, ReaderRevenueLinks, Subnav}
+import views.support.ContentLayout
+import play.api.libs.json
+import play.api.libs.json._
+import play.api.mvc.RequestHeader
+
+object InteractivesDotcomRenderingDataObject {
+
+  def makeDataObject()(implicit request: RequestHeader): DotcomRenderingDataModel = {
```

Comment:
Call it `mockDataObject` to make intention clear? Or rename the object to `InteractivesDotcomRenderingDataMock` and have this method be the `apply` method.

Reply:
Will do!